### PR TITLE
Fixing links on getting started guide

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: _posts/docker/guide/2015-06-10-getting-started.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ git clone git@github.com:codeship/ci-guide.git
 
 We're going to walk you through using Codeship to build, test, and deploy your Docker applications. Codeship uses a tool called Jet to turn your existing Docker apps and workflows into a seamless CI/CD process.
 
-The first thing you want to do is install Codeship Jet on your local machine. For Mac users, you can do this through Brew and Linux users can curl the Jet binary directly. [You can see more detailed instructions here.](https://codeship.com/documentation/docker/installation/)
+The first thing you want to do is install Codeship Jet on your local machine. For Mac users, you can do this through Brew and Linux users can curl the Jet binary directly. [You can see more detailed instructions here.](https://documentation.codeship.com/docker/installation/)
```
Comment (mlocher) [inline _posts/docker/guide/2015-06-10-getting-started.md]: Could we switch to using the `link` or `post_url` helpers. That way we don't have to fix the links again if we move those documents.

* http://jekyllrb.com/docs/templates/#link
* http://jekyllrb.com/docs/templates/#post-url

File: _posts/docker/guide/2016-07-27-getting-started-part-two.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ demo:
     TEST_TOKEN: Testing123
 ```
 
-This new `environment` directive creates a new environment variable in our build named `TEST_TOKEN`. Note that even though we're explicitly declaring our environment variables here, in a production application we'd actual prefer to [encrypt them](https://codeship.com/documentation/docker/encryption/).
+This new `environment` directive creates a new environment variable in our build named `TEST_TOKEN`. Note that even though we're explicitly declaring our environment variables here, in a production application we'd actual prefer to [encrypt them](https://documentation.codeship.com/docker/encryption/).
```
Comment (mlocher) [inline _posts/docker/guide/2016-07-27-getting-started-part-two.md]: Same as above, why not us the helper functions for the links?

File: _posts/docker/guide/2016-07-30-getting-started-part-five.md [hunk 3]
```diff
@@ -94,6 +94,6 @@ This tag tells Codeship to only run this tag on the master branch. You can imagi
 
 From here, there's still a ton more you can learn to optimize your builds, troubleshoot your problems and build more complex and productive workflows.
 
-We recommend [our blog](https://blog.codeship.com), [our documentation](https://codeship.com/documentation/), [our community forum](https://community.codeship.com), and [our webinars](http://resources.codeship.com/webinars) to keep learning more.
+We recommend [our blog](https://blog.codeship.com), [our documentation](https://documentation.codeship.com/), [our community forum](https://community.codeship.com), and [our webinars](http://resources.codeship.com/webinars) to keep learning more.
```
Comment (mlocher) [inline _posts/docker/guide/2016-07-30-getting-started-part-five.md]: The `site.url` variable contains the root URL of the site and is configured at https://github.com/codeship/documentation/blob/master/_config.yml#L2

**Comment by ethangj** [comment]
@mlocher Updated, lmk

File: _posts/docker/guide/2016-07-30-getting-started-part-five.md [hunk 4]
```diff
@@ -94,6 +94,6 @@ This tag tells Codeship to only run this tag on the master branch. You can imagi
 
 From here, there's still a ton more you can learn to optimize your builds, troubleshoot your problems and build more complex and productive workflows.
 
-We recommend [our blog](https://blog.codeship.com), [our documentation](https://codeship.com/documentation/), [our community forum](https://community.codeship.com), and [our webinars](http://resources.codeship.com/webinars) to keep learning more.
+We recommend [our blog](https://blog.codeship.com), [our documentation](({{ site.baseurl }}), [our community forum](https://community.codeship.com), and [our webinars](http://resources.codeship.com/webinars) to keep learning more.
```
Comment (mlocher) [inline _posts/docker/guide/2016-07-30-getting-started-part-five.md]: `site.baseurl` is only the root folder the site is published to, `site.url` is the full domain name (but, without the root folder).

See the `relative_url` and `absolute_url` filters available at http://jekyllrb.com/docs/templates/#filters

According to the docs the root page should be at `{{ "/" | absolute_url }}` or `{{ site.url }}{{ site.baseurl }}`

**Comment by ethangj** [comment]
@mlocher i'm sure this is not quite right yet, but how about now?

**Pull request merged**